### ISSUE TITLE
Adicionar Nuget

### ISSUE THREAD

**Issue by olavorn** [BoletoNet/BoletoNetCore#3]
Criar novo pacote nuget. Preferencialmente via CI-CD

**Comment by olavorn** [comment]
@carloscds poderías criar uma build para este projeto na tua conta do AppVeyor?

**Comment by olavorn** [comment]
Precisa alterar as badges também. Vou começar a mudar o nome do projeto pra facilitar na solution.

Até acho uma boa que a estrutura de namespaces volte a ficar:

BoletoNet (para o BoletoNetCore)
BoletoNet.Pdf (para o Projeto PDF)
BoletoNet.Testes (para o projeto de Testes). 

A outra opção seria BoletoNetCore. aí Precisariamos votar :)

**Comment by carloscds** [comment]
@olavorn Vou fazer isto.

**Comment by olavorn** [comment]
Vou mudar logo o nome ora boletoNetCore. Essa mudança vai acabar mechendo com todos os arquivos, então quanto mais cedo melhor. @marcosgerene, seria bom vc fazer logo um PR das suas alterações e pegar o meu PR em seguida, pois praticamente todos os arquivos vão mudar, em função da mudança do nome das pastas :/ Então deve dar muito conflito.

**Comment by olavorn** [comment]
![image](https://user-images.githubusercontent.com/3752407/61759589-34473d00-ad9f-11e9-843a-7740fb5542a6.png)

**Comment by carloscds** [comment]
@olavorn Temos um problema com o pacote, o nome BoletoNetCore já esta sendo usado no Nuget, alguma outra sugestão ?

**Comment by olavorn** [comment]
Putz,
Vou tentar falar com o @Tagliatti primeiro...

**Comment by carloscds** [comment]
@olavorn Blz.

**Comment by Tagliatti** [comment]
o q passa?

**Comment by carloscds** [comment]
@olavorn Pacote gerado e publicado no Myget. Só fica a questão do nuget agora.

**Comment by carloscds** [comment]
@Tagliatti Nós estamos migrando o projeto do boletonet2 para o Core, mas você já comecou isto. Gostaia de se juntar a nós ?

**Comment by Tagliatti** [comment]
Claro, só falar o q precisa.

**Comment by carloscds** [comment]
@Tagliatti Precisamos publicar o paote Nuget, mas o nome BoletoNetCore esta criado por você.

**Comment by Tagliatti** [comment]
me passa os usuários q eu adiciono como owner.

**Comment by carloscds** [comment]
@Tagliatti Adiciona o meu: carloscds

**Comment by carloscds** [comment]
@Tagliatti Valeu! 
@olavorn Testa o pacote que eu gerei no MyGet. Quando tivermos uma versão estável, podemos publicar no NuGet.

**Comment by olavorn** [comment]
Vou testar hj a noite e aviso

**Comment by carloscds** [comment]
@olavorn Conseguiu testar o pacote do myget?

**Comment by olavorn** [comment]
Não, na verdade ia pedir pra vc rodar esse PR antes, pois com as modificacoes lá possibilitam que eu teste em um sistema nosso.

**Comment by olavorn** [comment]
![image](https://user-images.githubusercontent.com/3752407/61912322-282ebd00-af10-11e9-97a3-d7b1f2a5734e.png)

**Comment by carloscds** [comment]
@olavorn Viu a mensagem em amarelo? Você tem que adicionar no Config do Nuget.

**Comment by olavorn** [comment]
@carloscds , hum verdade. Eu tinha copiado direto da Url do MyGet, mas o que eles não falam é que tem que rodar esse comando antes: 
```
nuget sources add -Name BoletoNetCoreMyGet -Source https://www.myget.org/F/boletonetcorebuild/api/v3/index.json
```

**Comment by carloscds** [comment]
@olavorn Isto mesmo, tem que adicionar nos feeds.

**Comment by hsbtiago** [comment]
Bom dia senhores! Mal conheço vocês mas já considero pakas! xD

Estou implementando o boletonetcore num projeto aqui do trabalho =D

Tem algo específico que eu possa fazer para ajudar na atualização do pacote do nuget? 

Vlw =D

**Comment by carloscds** [comment]
@hsbtiago Fala Tiago, pode nos ajudar contribuindo com o projeto ;-)

**Comment by olavorn** [comment]
Opa @hsbtiago ! Estamos fazendo algumas atualizações mais radicais em relação aos outros projetos aproveitando o fato de ser um NOVO projeto :)

Basicamente as premissas que todos partimos e concordamos estão lá de README. Com isso em mente, vamos fazer as mudanças necessárias para dar uma identidade ao projeto e receber adeptos justamente pelas coisas que nos diferenciam dos outros projetos, ainda assim aproveitando todo o know how daqueles que nos trouxeram até aqui. 👍

**Comment by olavorn** [comment]
@carloscds o pacote ficou ok, mas não consegui fazer os testes funcionarem :(

**Comment by olavorn** [comment]
@igorocampos consegues [atender](https://github.com/BoletoNet/BoletoNetCore/issues/33#issuecomment-534159682) ?

**Comment by carloscds** [comment]
@olavorn Posso fazer, pois envolve o processo de build e publicacao.
